server/channels/api4/channel_bookmark_test.go

```diff
@@ -1375,7 +1375,7 @@ func TestListChannelBookmarksForChannel(t *testing.T) {
 
 	// an open channel for which the guest is a member but the basic
 	// user is not
-	onlyGuestChannel := th.CreateChannelWithClient(th.SystemAdminClient, model.ChannelTypeOpen)
+	onlyGuestChannel := th.CreateChannelWithClient(th.SystemAdminClient, model.ChannelTypePrivate)
```

Comment:
In general, I'd prefer if we added a new test case rather than changing an existing one.